### ISSUE TITLE
Dark theme: google chrome active tab not very distinguishable

### ISSUE THREAD

**Issue by chrisjbillington** [ubuntu/communitheme#568]
Using the dark theme, I find myself having to scan my eyes carefully over the tab bar in Chrome to see which tab is active - particularly when Chrome is maximised the active tab doesn't really jump out and has to be hunted for. Here is a screenshot:

![screenshot](https://user-images.githubusercontent.com/1044087/41633313-9db89034-7481-11e8-827f-176937b259ab.png)

Firefox on the other hand has the active tab much lighter, which jumps out and is easy to see:

![ffox_ss](https://user-images.githubusercontent.com/1044087/41633328-b01a4eca-7481-11e8-8fe9-742ec4eccdb0.png)

**Comment by clobrano** [comment]
Hi @chrisjbillington, thanks for reporting this. Full dark variant is not our first priority at the moment, at least not for applications that are not using it by default, but we will get to this asap

**Comment by chrisjbillington** [comment]
No worries!

**Comment by Feichtmeier** [comment]
![image](https://user-images.githubusercontent.com/15329494/41929827-9f5ebc5e-7979-11e8-8971-fd748edb87ee.png)

We can darken the headerbar. Which gives a bit more contrast over all. If this fix is okay for everyone

**Comment by Feichtmeier** [comment]
@clobrano failing to find a way to style that checked tab or the regular ones

**Comment by chrisjbillington** [comment]
Darker header helps distinguish tabs and header, which is nice, but yes, is separate to the reported issue.

Changing the font colour or something else about the tabs would also help make them more distinguishable, if there is any access to any other details that differ between active and inactive tabs. Colour would be the best but if it can't be done then it can't be done.

**Comment by Feichtmeier** [comment]
Nvm, changing the header color creates more problems than it solves.

**Comment by Feichtmeier** [comment]
I have really no idea how to style chrome's tabs. Since they will change anyways with the next big release (they will be.... round and... different :dancer: ) I close this now.
Thanks for reporting though.

**Comment by chrisjbillington** [comment]
With chrome's new UI, this is still a problem.

Here is yaru:

![image](https://user-images.githubusercontent.com/1044087/50205543-b6602500-03bc-11e9-9188-f3df40eeab0b.png)

Here is yaru-dark:

![image](https://user-images.githubusercontent.com/1044087/50205521-9e88a100-03bc-11e9-907b-946617e9052d.png)

Though it's not much better on other dark themes, so this isn't unique to yaru. Here is adwaita-dark for example:

![image](https://user-images.githubusercontent.com/1044087/50205786-8d8c5f80-03bd-11e9-8866-a37a7b4e5338.png)

**Comment by clobrano** [comment]
Ad of today there isn't a clear way to force a different style on chrome. It basically takes some color from our palette

**Comment by chrisjbillington** [comment]
Perhaps I ought to report the bug against Chrome/chromium then.

**Comment by clobrano** [comment]
At least to get an hint about the css targets to use

**Comment by WaldemarH** [comment]
Yeah the selected tab needs to be highlited as it's really a pain to use. :/

**Comment by Feichtmeier** [comment]
@WaldemarH 
I am sorry, but I strongly disagree:

![image](https://user-images.githubusercontent.com/15329494/58365916-502b8100-7ecb-11e9-9499-dcbed4ce2b0a.png)

The selected / non-selected tabs are perfectly distinguishable

**Comment by WaldemarH** [comment]
Maybe if you have 3 tabs opened and are a young person with monitor brightness all the way up, then I agree with you. But when you are programming for last 30+ years, have dimmer screen, using programmers (anti-blue light) glasses, then I can assure you that you are FAAAAAR from seeing which tab is selected.

I've done some digging and for now I've found that in Vivaldi you can just set your custom.css and tadaaa you have your custom tab color. Now I don't know if Chrome already has some option to modify its look through custom css, but that would be all I need. So a person can adjust the preferences to his/her needs... here is an example of a simple 3 line mod in Vivaldi.

#browser .tab-position .tab.active {
  background-color: white;	/*var(--colorAccentBg);*/
  color: black;	/*var(--colorAccentFg);*/
  fill: white;	/*var(--colorAccentFg);*/
}

![tabs](https://user-images.githubusercontent.com/5547441/58367536-f03fd500-7ee0-11e9-81eb-8f16c701be2e.jpg)

**Comment by Pixelbits-nl** [comment]
Any progress on this? Active tabs in dark mode are virtually indistinguishable from inactive tabs.

**Comment by clobrano** [comment]
As per this comment https://github.com/ubuntu/yaru/issues/568#issuecomment-448499491

This is not something we can do much about it and the ticket's closed

**Comment by WaldemarH** [comment]
@Pixelbits-nl You can try with a different theme or just (as I did) simply go to Vivaldi that has the same JS engine, but much, much superior customization.

Whenever the new version arrives I just copy:
.tab-position .tab {
  background-color: #888;
  border: 1px solid #FFF;
  border-radius: 3px !important;
}

#browser .tab-position .tab.active {
  background-color: white;	/*var(--colorAccentBg);*/
  color: black;	/*var(--colorAccentFg);*/
  fill: black;	/*var(--colorAccentFg);*/
}
to the end of the custom.css file and I get:

![image](https://user-images.githubusercontent.com/5547441/60000148-e508d200-9663-11e9-93e8-a5f3975dc9e8.png)

I use gray for un-active tabs (not black because most of the icons are made for white background) and then I define active tab as a fully white (CSS) and wide expanded(setting in Vivaldi) tab.
Hope that helps. :)

**Comment by Pixelbits-nl** [comment]
Thanks - will do :)

**Comment by purplejacket** [comment]
Come Chrome devs, this is a pretty annoying issue.  Steve Jobs would not sit still for this one.  Find a way.

**Comment by JDea** [comment]
Good alternative theme with easier to see tabs.
https://chrome.google.com/webstore/detail/material-dark/npadhaijchjemiifipabpmeebeelbmpd?hl=en

**Comment by DavidBruunLie** [comment]
Thanks, JDea - that plugin solves it for me. Google - If your'e still listening to this thread - this is how the active-tab should be solved natively in dark mode. Just claiming that you cannot do anything about it is pure arrogance.

**Comment by johnyradio** [comment]
This is driving me nuts as well. i prefer dark theme to save my eyes, but i'm forced to use light theme because i cannot distinguish the active tab. 

Imo, enough people here have complained that the issue shouldn't be closed.

**Comment by clobrano** [comment]
@johnyradio, I'm sorry for this, but the problem doesn't seem to have a solution on our side

> As of today there isn't a clear way to force a different style on chrome. It basically takes some color from our palette

**Comment by johnyradio** [comment]
If themes are supported, that means tab-styling is supported.

Could be a theme or extension which does nothing except style the active
tab.

>

**Comment by clobrano** [comment]
That's would be very good, are you willing to help?

**Comment by johnyradio** [comment]
i'm no expert, but here's what i've found so far.

<https://youtu.be/frQHKz3wnew>
<https://sites.google.com/site/gsugsa/google-apps/google-chrome/how-to-create-a-theme>
<https://developer.chrome.com/extensions/themes>

**Comment by clobrano** [comment]
Thank you for the research, but I'm afraid there's a misunderstanding. Yaru is a gtk theme, what you have found is a way to create a theme for chrome only.

**Comment by johnyradio** [comment]
my bad.
but maybe blink theming is a better (or only) way to do it.
blink =  chrome, chromium, opera, vivaldi, etc

**Comment by clobrano** [comment]
It's a way to do it, but it's a different project. You won't have your chrome theme installed automatically with Yaru, you'd need to install it separately

**Comment by johnyradio** [comment]
i see what you mean.
But if it's the only way...
:)

**Comment by Feichtmeier** [comment]
@johnyradio 
This does not happen if you use the dark theme that comes with master currently:
![grafik](https://user-images.githubusercontent.com/15329494/65040092-ea6c7480-d953-11e9-9cb0-88eed3eae29c.png)

**Comment by johnyradio** [comment]
@ Feichtmeier
My apologies, i'm just using Opera browser default Dark mode, not the 
Yaru theme.

But your image shows the problem i'm seeing. Depending on display 
hardware, angle of view, and brightness setting, the active tab can't 
be distinguished. In my scenario, that's usually.

The active tab needs much stronger difference (could be a different 
background color, different font color, different font style, anything)

**Comment by WaldemarH** [comment]
@johnyradio Just start using Vivaldi and all the issues will be gone. You can change the design as much as you wish. Currently I'm using Chrome only for debugging and nothing else anymore, Vivaldi is perfect for my browser needs.

**Comment by johnyradio** [comment]
Thx, I've tried Vivaldi, didn't like it. Tried all  blink browsers on
Linux. Opera consistently gives me best performance and UE. Theming Opera
seems like a good solution.

>

**Comment by WaldemarH** [comment]
I've used opera from probably beginning.. like 1.x, but lately it's too much of a Chrome follower.
Now I've tested Vivaldi a few years back and it was not good enough.. video playback didn't work, pages didn't work,... but after this debacle I've tried it again and all is working perfectly. So now I use Chrome for debugging of one customer application, Opera for debugging my application, Vivaldi for my daily browsing needs and Edge for morning news reading. :)  .. have also Firefox, but I simply can't use it.. everything feels weird.
